lib/Screens/Group/Add_Expense/add_expense.dart

```diff
@@ -0,0 +1,408 @@
+import 'package:currency_picker/currency_picker.dart';
+import 'package:flutter/material.dart';
+import 'package:splitwise/Routes/navigation_functions.dart';
+
+import '../../../CommonMethodClass/theme_data.dart';
+import '../../../Utils/colors.dart';
+import '../../../Utils/common_strings.dart';
+
+class AddExpense extends StatefulWidget {
+  const AddExpense({Key? key}) : super(key: key);
+
+  @override
+  State<AddExpense> createState() => _AddExpenseState();
+}
+
+class _AddExpenseState extends State<AddExpense> {
+  TextEditingController descriptionController = TextEditingController();
+  TextEditingController amountController = TextEditingController();
+  Currency? currentSymbol;
+
+  @override
+  void dispose() {
```

Comment:
Please follow this order

initState
didChangedependency
Build
dispose

Never break this order.